app/Models/Reference/Country.php

```diff
@@ -28,4 +28,9 @@ public function teams(): HasMany
     {
         return $this->hasMany(Team::class);
     }
+
+    public function xlsformModuleVersion(): HasMany
```

Comment:
Just for consistency, I'd use the plural for HasMany or BelongsToMany relationships.